arch/arm/src/sama5/Kconfig

```diff
@@ -3246,11 +3247,12 @@ config SAMA5_ADC_ANARCH
 	default n
 	---help---
 		This option allows you to select different gain, offset, and single
-		vs. differential modes for each channel.
+		vs. differential modes for each channel, depending on ARCH.
 
 if SAMA5_ADC_ANARCH
 
 menu "Channel gain"
+    depends on ARCH_CHIP_SAMA5D
```

Comment:
TABs

Reply:
This has the same indentation as, say, line 816 and 925. What is wrong?